### ISSUE TITLE
Hashtag exclusion filters

### ISSUE THREAD

**Issue by sivy** [pixelfed/pixelfed#4547]
It would be great to be able to consruct a query with somethig like `not:` and be able to filter out posts with a one or more hashtags. A classic example would be filtering out `#nsfw` (though most platforms now force you to _include_ those rather than exclude). 

I am an artist and would like to be able to, for example, subscribe to a search like: `#abstractart not:#digital`

**Comment by Satchmo35** [comment]
And the sooner the better...not overly useful to browse when I do not want to see about 90 per cent of the pics I currently do.

**Comment by MaiaDereva** [comment]
I agree. Coming from Mastodon, I miss this simple feature on Pixelfed a lot!
